packages/office-ui-fabric-react/src/components/FocusTrapZone/examples/FocusTrapZone.Box.Click.Example.tsx

```diff
@@ -16,51 +16,33 @@ export class FocusTrapZoneBoxClickExample extends React.Component<{}, IFocusTrap
   private _toggle = React.createRef<IToggle>();
 
   public render() {
-    return (
-      <div>
-        {this.state.useTrapZone ? (
-          <FocusTrapZone isClickableOutsideFocusTrap={true} forceFocusInsideTrap={false}>
-            {this._internalContents()}
-          </FocusTrapZone>
-        ) : (
-          this._internalContents()
-        )}
-      </div>
-    );
-  }
-
-  private _internalContents() {
     const { useTrapZone } = this.state;
 
     return (
-      <Stack
-        horizontalAlign="start"
-        tokens={{ childrenGap: 15 }}
-        styles={{
-          root: { border: `2px dashed ${useTrapZone ? '#ababab' : 'transparent'}`, padding: 10 }
-        }}
-      >
-        <Toggle
-          label="Use trap zone"
-          componentRef={this._toggle}
-          checked={useTrapZone}
-          onChange={this._onFocusTrapZoneToggleChanged}
-          onText="On (toggle to exit)"
-          offText="Off"
-        />
-        <TextField label="Input inside trap zone" styles={{ root: { width: 300 } }} />
-        <Link href="https://bing.com">Hyperlink inside trap zone</Link>
-      </Stack>
+      <FocusTrapZone disabled={!useTrapZone} isClickableOutsideFocusTrap={true} forceFocusInsideTrap={false}>
+        <Stack
+          horizontalAlign="start"
+          tokens={{ childrenGap: 15 }}
+          styles={{
+            root: { border: `2px dashed ${useTrapZone ? '#ababab' : 'transparent'}`, padding: 10 }
```

Comment:
Suggestion: Can you move these styles/tokens to their own `consts`?

Reply:
May be a difference of opinion, but I don't really see the need to use consts if the style is only used once

Reply:
I can see arguments for both sides. Examples should be self-contained, but it's good to show best practices as well. Props shouldn't be mutating unnecessarily either, and inline objects will mutate every time.